cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -68,11 +68,17 @@ spec:
       - name: rook-ceph-operator
         image: rook/ceph:master
         args: ["ceph", "operator"]
+        ports:
+          - containerPort: 8443
```

Comment:
8443 is the port used by the dashboard... can you use another default port?

Reply:
It's a different service IP so it doesn't really matter IIRC unless everything runs on the same machine.

Reply:
yep, this could affect tests/labs/demos using one machine, taking into account that we have several thousand of other possibilities... why to temp fate?

Reply:
We have to allow things to work on a single machine by default. Assuming we can register any endpoint, let's pick a less common port, but still make it configurable here.

Reply:
Fixed. Changed port to 8079